src/README.md

```diff
@@ -10,7 +10,7 @@ In the vagrant:
     $ sudo virt-host-validate
     $ sudo su
     # systemctl start virtlogd.socket
-    # systemctl start libvirtd.service
+    # systemctl start libvirtd.socket or # systemctl start virtqemud.socket
```

Comment:
If you want to comment out the alternative to allow copy pasting into terminal put the # before the or.